scripts/populateDb.ts

```diff
@@ -0,0 +1,60 @@
+import { ObjectID } from 'mongodb';
+import faker from 'faker';
+import institutions from '../src/client/assets/data/institutions.json';
+import {
+	ApplicationStatus,
+	ShirtSize,
+	Gender,
+	DietaryRestriction,
+	Race,
+	UserType,
+	HackerDbObject,
+} from '../src/server/generated/graphql';
+import { initDb } from '../src/server/models';
+
+require('dotenv').config();
+
+const getRandom = (max: number): number => Math.floor(Math.random() * max);
+const NUM_HACKERS = 800;
+
+const generateHacker: () => HackerDbObject = () => {
+	const fn = faker.name.firstName();
+	const ln = faker.name.lastName();
+	return {
+		_id: new ObjectID(),
+		createdAt: new Date(),
+		dietaryRestrictions: [Object.values(DietaryRestriction)[getRandom(7)]],
+		email: faker.internet.email(),
+		firstName: fn,
+		gender: Object.values(Gender)[getRandom(7)],
+		gradYear: getRandom(4) + 2019,
+		lastName: ln,
+		logins: [],
+		majors: [],
+		modifiedAt: new Date().getTime(),
+		phoneNumber: faker.phone.phoneNumber(),
+		preferredName: fn,
+		race: [Object.values(Race)[getRandom(6)]],
+		school: institutions[getRandom(1430)],
+		secondaryIds: [],
+		shirtSize: Object.values(ShirtSize)[getRandom(6)],
+		status: Object.values(ApplicationStatus)[getRandom(7)],
+		userType: UserType.Hacker,
+	};
+};
+
+const addHackers = async (): Promise<void> => {
+	const models = await initDb();
+	const newHackers: HackerDbObject[] = [];
+	for (let i = 1; i <= NUM_HACKERS; i += 1) {
```

Comment:
Should we enable the eslint rule against plusplus?

Reply:
feel free to make a PR

Reply:
I'm strongly against this rule - but Matt seems for it? Idk

Reply:
Either way is fine with me, I don't have a preference.